### ISSUE TITLE
Start_climate fails

### ISSUE THREAD

**Issue by ThomasEAK** [Hyundai-Kia-Connect/kia_uvo#500]
Kia Rio +Tech Package
USA

All calls to start_climate error in UI or automation, and this is in the logs:
error: unknown error response {"status":{"statusCode":1,"errorType":1,"errorCode":9001,"errorMessage":"Incorrect request payload format"}}
error: unknown error response {"status":{"statusCode":1,"errorType":1,"errorCode":1005,"errorMessage":"Invalid vehicle for current session"}}

This includes automations that worked prior updating. I've rebuilt/tested in dev space via UI and YAML, and no combination of parameters succeeded. I've removed and readded my car/account in case my vehicle ID was wrong.

**Comment by cdnninja** [comment]
A few things to try.  

1. Could you confirm lock/unlock work?  
2. Could you enable debug logs as outlined in the main page and post the logs it produces?  Can you send the YAML parameters you are sending?

**Comment by cdnninja** [comment]
Please also confirm the version you are on.  I did some fixes for this in the last week.

**Comment by ThomasEAK** [comment]
Running current version 2.0.48
Errors started after a recent update at the beginning of this week. I may have been a version or two behind though.
Unlock and lock with service and lock entity both working perfectly.

YAML
service: kia_uvo.start_climate
data:
  device_id: c5c3c6c024e74d17267af529ec6494b6
  duration: 10
  temperature: 72
  heating: "0"
  defrost: false


When I went back into HA to copy out the code, I found the whole service listed unavailable. I went in and did a force reload on the integration and as of now everything including climate start is working. I restarted HA multiple times after updating the integration but I've never had to manually tell it to reload from the system before.
